packages/components/src/spectrum/picker/Picker.tsx

```diff
@@ -89,14 +110,21 @@ export function Picker({
           onSelectionChange) as NormalizedSpectrumPickerProps['onSelectionChange']
       }
     >
-      {({ content, textValue }) => (
-        <Item textValue={textValue === '' ? 'Empty' : textValue}>
-          <PickerItemContent>{content}</PickerItemContent>
-          {tooltipOptions == null || content === '' ? null : (
-            <Tooltip options={tooltipOptions}>{content}</Tooltip>
-          )}
-        </Item>
-      )}
+      {itemOrSection => {
+        if ('items' in itemOrSection) {
```

Comment:
Should write a type predicate for this, e.g. `isPickerSectionItem`

Reply:
done